container/raw/factory.go

```diff
@@ -29,6 +30,7 @@ import (
 )
 
 var dockerOnly = flag.Bool("docker_only", false, "Only report docker containers in addition to root stats")
+var dockerOnlyPrefixWhitelist = flag.String("docker_only_prefix_whitelist", "", "A comma-separated list of cgroup path prefix that needs to be collected when docker-only is specified")
```

Comment:
can we call this rawPrefixWhitelist?  I am hoping we can move away from `dockerOnly`, and instead use a whitelist of handlers in the future.